packages/injectable/react/src/withInjectables/withInjectables.test.jsx

```diff
@@ -166,6 +166,33 @@ describe('withInjectables', () => {
     expect(rendered.baseElement).toMatchSnapshot();
   });
 
+  it('given component and sync dependencies, when rendered without provider, renders to null', () => {
```

Comment:
Would it be more justifiable to throw an explanatory error about missing `di` in context? I mean, that should always be a case of misuse, right?

Reply:
Or are there justified, yet unfair, reasons for this to momentarily be the case? Why do we discover it only now?

Reply:
> I don't know, why we are just finding it now.

I consider it likely that the misdesign to cause missing `di` is somewhere else, and to silently render `null` in response is a harmful hack around that. Hiding from unexpected circumstances is a questionable policy in software, as it guides towards not understanding what is the real root of problem.

> I know that you seem to be abusing the API according to the documentation and that context consumers should be written in a way that is resliant against being rendered outside of a provider.

Do you have link to API-documentation where the requirement for said resiliency is explained? Common sense would indicate that missing `di`
> should always be a case of misuse, right?

Reply:
> I consider it likely that the misdesign to cause missing `di` is somewhere else...

This turned out to be the case, closing PR :)